jenkins_job_wrecker/modules/triggers.py

```diff
@@ -38,18 +40,211 @@ def reversebuildtrigger(top, parent):
         if child.tag == 'upstreamProjects':
             reverse['jobs'] = child.text
         elif child.tag == 'threshold':
-            pass    # TODO
+            pass  # TODO
         elif child.tag == 'spec':
-            pass    # TODO
+            pass  # TODO
         else:
             raise NotImplementedError('cannot handle reverse trigger '
                                       'setting %s' % child.tag)
 
     parent.append({'reverse': reverse})
 
 
+def process_file_paths(attribute):
```

Comment:
I'd prefix the function with 'gerrit_'

Reply:
I will also make the funcs private with double underscore prefix.